tests/network.md

```diff
@@ -485,3 +485,24 @@ EPIPE:
     (fun () -> Eio.Flow.shutdown a `Receive);;
 - : unit = ()
 ```
+
+## Addrinfo (Linx and Libuv)
```

Comment:
```suggestion
## Addrinfo (Linux and Libuv)
```

Reply:
Removed the redundant line and luv test.